Eplant/tutorial.md

```diff
@@ -0,0 +1,227 @@
+
+### Atoms: 
+Basic units of state. An atom holds a piece of state that can be read from and written to.
+### Hooks: 
+useAtom is used to interact with atoms, allowing components to access and update state.
+### Simplicity:
+Minimal boilerplate and API surface, making it easy to understand and use.
+### Reactivity: 
+Components automatically re-render when atom state changes.
+
+## When you create a new `view`
+If you are going to a build new View, `GetStartedView` (`Eplant/views/GetStartedView/index.tsx`) is the right one to look at.
+
+First, create `index.tsx` by defining and exporting a View configuration object that uses the `GetStartedView` component.
+
+### Define GetStartedView Object:
+
+**name**: Set a descriptive name for the view ("Get started").  
+**component**: Assign the GetStartedView component.  
+**getInitialData**: Define a method to fetch initial data (returns null in this case).  
+**id**: Assign a unique identifier ("get-started").  
+**icon**: Specify an icon (HomeOutlinedIcon) for the view.  
+
+
+Next, create the `GetStartedView` component that serves as an introductory view, guiding users on how to interact with the application.
+
+### Define GetStartedView Component:
+
+**Theme**: Use useTheme hook to access the theme for styling.  
+**Render Component**:  
+Display introductory text about the application.
+Provide instructions using Material-UI's Stack and Typography components.
+Create a Grid layout to display multiple Tile components for each view, filtered by the presence of description and thumbnail.
+
+
+If you need an additional component, follow `Tile.tsx`. This is used in `component.tsx`.
+
+Create a UI component ( `Tile.tsx`) that displays a view's information and allows users to activate a specific gene and view when interacting with it.
+
+### Define Tile Component:
+
+**Props**: Accept a view object which includes information about the view.  
+**State and Hooks**:
+Use useGeneticElements, useSetActiveGeneId, and useSetActiveViewId hooks to manage genetic elements and active view/gene states.  
+**`setView` Function**:
+Fetch a specific gene (AT3G24650) using the arabidopsis.api.searchGene method.
+Set this gene as the active gene and update the active view.  
+**Render Component**:  
+Use Material-UI's Card component to structure the tile, including CardMedia, CardContent, and CardActions.
+Display the view's thumbnail, name, and description.
+Add a button to activate the view and gene when clicked.
+
+
+### Summary
+1. Create `index.tsx`: An object that configures the view with the GetStartedView component and relevant metadata.
+2. Create `component.tsx`: A component that provides a guide for getting started and displays a grid of Tile components.
+3. Create `Tile.tsx`: A component for displaying individual view tiles with an interactive button to set active genes and views.
+
+
+
+## Eplant2  
```

Comment:
This is a stupid approach, but can you add a section here that clarifies that using the Developer's console from the browser and poking around ePlant2 is the way that a developer can find the BAR APIs that they will need to call?